package.json

```diff
@@ -29,7 +29,8 @@
     "build": "tsc",
     "build:clean": "shx rm -rf ./dist ./coverage ./.nyc_output",
     "lint": "tslint --project .",
-    "test": "nyc mocha --config .mocharc.json \"src/**/*.spec.ts\"",
+    "mocha": "nyc mocha --config .mocharc.json \"src/**/*.spec.ts\"",
```

Comment:
added a separate script for mocha so it's still easy to run this command in isolation.